### PR TITLE
Update Rubocop to be an executable

### LINKED ISSUE #2066: Version 0.32.0 broke Rubocop formatting when .rubocop.yml file does not already exist
# Description

After the upgrade to 0.32.0 I can't format ruby files anymore. It shows the running message, but does nothing to the code.

If I downgrade to 0.30.9 it works again.

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/escobera/84f2f0218297c5dd8c9f5b5b0d9c7e2a

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

### PR DESCRIPTION
**Pull request by szeck87**
### What does this implement/fix? Explain your changes.
* Updates Rubocop to be a formal Atom Beautify executable
* Only passes cwd if a .rubocop.yml file is present
...

### Does this close any currently open issues?
Closes #2066 
...

### Any other comments?

...

### Checklist

Check all those that are applicable and complete.

- [X] Merged with latest `master` branch
- [X] Regenerate documentation with `npm run docs`
- [ ] Add change details to `CHANGELOG.md` under "Next" section
- [X] Added examples for testing to [examples/ directory](examples/)
- [X] Travis CI passes (Mac support)
- [X] AppVeyor passes (Windows support)

### CODE REVIEW

File: src/beautifiers/rubocop.coffee [hunk 1]
```diff
@@ -51,9 +63,9 @@ module.exports = class Rubocop extends Beautifier
       rubocopArguments.push("--config", tempConfig) if tempConfig?
       @debug("rubocop arguments", rubocopArguments)
 
-      @run(rubocopPath, rubocopArguments, {
```
Comment (Glavin001) [inline src/beautifiers/rubocop.coffee]: `rubocopPath` should be deprecated and/or removed from supported options.
See https://github.com/Glavin001/atom-beautify/blob/5f968cb9f33ebd235471aaa7ad271c8292fe190e/src/beautifiers/php-cs-fixer.coffee#L91-L93 for example.

Comment (Glavin001) [reply src/beautifiers/rubocop.coffee]: May also want to be backward compatible for a couple versions by falling back to `@run(rubocopPath` when `rubocopPath` is set.

**Comment by Glavin001** [comment]
@bbugh @escobera I appreciate your review of this PR since it is applicable to you.

**Comment by Glavin001** [comment]
Awesome, thank you for testing @escobera !

**Comment by bbugh** [comment]
@Glavin001 covered my comments already. Looks great @szeck87, glad that #2066 was an easy fix for @escobera!

**Comment by szeck87** [comment]
@escobera @bbugh could you please test once more (even if you have already)?  I made another change and want to make sure it still works for both of you.

**Comment by bbugh** [comment]
Seems to be working. I tested it 

1) **with** rubocop path specified in options
2) **without** rubocop path in options (empty, default)
3) no `.rubocop.yml` in tree structure **with** `Indent Size` specified
3) no `.rubocop.yml` in tree structure **without** `Indent Size` specified
4) `.rubocop.yml` in user home folder **and** current project folder
5) `.rubocop.yml` in user home folder **but not** current project folder

All seem to be working just fine. 👍

**Comment by Glavin001** [comment]
Thank you all for your input! I will post back here once the fix has been published.

**Comment by szeck87** [comment]
Thanks for the thorough testing @bbugh.

**Comment by Glavin001** [comment]
Published to v0.32.2

**Pull request merged**